index.html

```diff
@@ -11,22 +11,200 @@
+                <ul class="language">
+                    <li><button type="button">Ruby on rails</button></li>
+                    <li><button type="button">css</button></li>
+                    <li><button type="button">Java Script</button></li>
+                    <li><button type="button">HTML</button></li>
+                </ul>
+                <button class="projects" type="button">See projects</button>
+            </div>
+            <div>
+                <img class="img" src="images/plan.png" alt="planholder">
+                <h2>Multi-Post Stories Gain+Glory</h2>
+                <ul class="language">
+                    <li><button type="button">Ruby on rails</button></li>
+                    <li><button type="button">css</button></li>
+                    <li><button type="button">Java Script</button></li>
+                    <li><button type="button">HTML</button></li>
+                </ul>
+                <button class="projects" type="button">See projects</button>
+            </div>
+            <div>
+                <img class="img" src="images/plan.png" alt="planholder">
+                <h2>Multi-Post Stories Gain+Glory</h2>
+                <ul class="language">
+                    <li><button type="button">Ruby on rails</button></li>
+                    <li><button type="button">css</button></li>
+                    <li><button type="button">Java Script</button></li>
+                    <li><button type="button">HTML</button></li>
+                </ul>
+                <button class="projects" type="button">See projects</button>
+            </div>
+            <div>
+                <img class="img" src="images/plan.png" alt="planholder">
+                <h2>Multi-Post Stories Gain+Glory</h2>
+                <ul class="language">
+                    <li><button type="button">Ruby on rails</button></li>
+                    <li><button type="button">css</button></li>
+                    <li><button type="button">Java Script</button></li>
+                    <li><button type="button">HTML</button></li>
+                </ul>
+                <button class="projects" type="button">See projects</button>
+            </div>
+            <div>
+                <img class="img" src="images/plan.png" alt="planholder">
+                <h2>Multi-Post Stories Gain+Glory</h2>
+                <ul class="language">
+                    <li><button type="button">Ruby on rails</button></li>
+                    <li><button type="button">css</button></li>
+                    <li><button type="button">Java Script</button></li>
+                    <li><button type="button">HTML</button></li>
+                </ul>
+                <button class="projects" type="button">See projects</button>
+            </div>
+        </div>
+            </section>
+            <section class="AboutMeHeader">
+                <div class="resume">
+                <h3 class="me">About me</h3>
```

Comment:
- [ ] Please consider adding the two background images to this section, so it is as close as possible to the Figma design

|Your design|Figma design|
|-|-|
|![Screenshot_20220609_163756](https://user-images.githubusercontent.com/25227022/172887943-0d6dfaa5-9e4a-4787-82fb-37b18a80bfd7.png)|![Screenshot_20220609_163944](https://user-images.githubusercontent.com/25227022/172887987-bb756057-64f4-42f7-8703-f9042c9dc919.png)|